sycl/include/sycl/handler.hpp

```diff
@@ -3382,18 +3383,20 @@ class __SYCL_EXPORT handler {
   template <typename PropertiesT>
   std::enable_if_t<
       ext::oneapi::experimental::is_property_list<PropertiesT>::value>
-  throwIfGraphAssociatedAndKernelProperties() {
+  throwIfGraphAssociatedAndKernelProperties() const {
     if (!std::is_same_v<PropertiesT,
                         ext::oneapi::experimental::detail::empty_properties_t>)
-      throwIfGraphAssociated<ext::oneapi::experimental::detail::SyclExtensions::
-                                 sycl_ext_oneapi_kernel_properties>();
+      throwIfGraphAssociated(
+          ext::oneapi::experimental::detail::UnsupportedGraphFeatures::
+              sycl_ext_oneapi_kernel_properties);
   }
 
   // Set value of the gpu cache configuration for the kernel.
   void setKernelCacheConfig(sycl::detail::pi::PiKernelCacheConfig);
 
-  template <ext::oneapi::experimental::detail::SyclExtensions ExtensionT>
-  void throwIfGraphAssociated();
+  void throwIfGraphAssociated(
+      ext::oneapi::experimental::detail::UnsupportedGraphFeatures Feature)
+      const;
```

Comment:
Reckon this will need an update to the `sycl/test/abi/sycl_symbols_linux.dump` test. I can push a windows test update.